helm/reana/templates/reana-server.yaml

```diff
@@ -101,6 +101,8 @@ spec:
           - name: DASK_ENABLED
             value: {{ .Values.dask.enabled | quote }}
           {{- if .Values.dask.enabled }}
+          - name: DASK_AUTOSCALER_ENABLED
+            value: {{ .Values.dask.autoscaler_enabled | quote }}
```

Comment:
BTW when testing the autoscaler PR set with the auto scaler turned off:

```
  autoscaler_enabled: false
  cluster_default_number_of_workers: 4
```

I'm seeing a problem with stopping workflows:

```console
$ rc run -w calver
...
$ rc stop -w calver --force
==> ERROR: Cannot stop workflow calver:
(404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '4c08dda6-899b-4da2-bdf7-2c2c76006ce6', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': '6b060355-5047-4214-899c-623e82a0c569', 'X-Kubernetes-Pf-Prioritylevel-Uid': '498afda6-b2ba-4561-9313-9bda4f75865d', 'Date': 'Fri, 01 Nov 2024 14:13:25 GMT', 'Content-Length': '374'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"daskautoscalers.kubernetes.dask.org \"dask-autoscaler-reana-run-dask-ce1f4c15-36ff-42fc-b320-b5f277322db7\" not found","reason":"NotFound","details":{"name":"dask-autoscaler-reana-run-dask-ce1f4c15-36ff-42fc-b320-b5f277322db7","group":"kubernetes.dask.org","kind":"daskautoscalers"},"code":404}
```

The worklow is stopped, the Dask workers are stopped, but the job pod fails with:

```console
$ k logs reana-run-job-42c23954-93d7-4524-8e9b-daa28e8c1fcd-5p4fg
...
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 167, in submit
    raise RuntimeError('cannot schedule new futures after shutdown')
RuntimeError: cannot schedule new futures after shutdown
Exception ignored in: <function Client.__del__ at 0x7ff1dc223100>
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/distributed/client.py", line 1736, in __del__
  File "/usr/local/lib/python3.11/site-packages/distributed/client.py", line 1991, in close
  File "/usr/local/lib/python3.11/site-packages/distributed/utils.py", line 434, in sync
TimeoutError: timed out after 60 s.

$ k logs reana-run-batch-ce1f4c15-36ff-42fc-b320-b5f277322db7-qhzq2
2024-11-01 14:14:31,967 | root | MainThread | INFO | Workflow ce1f4c15-36ff-42fc-b320-b5f277322db7 finished. Files available at /var/reana/users/00000000-0000-0000-0000-000000000000/workflows/ce1f4c15-36ff-42fc-b320-b5f277322db7.
```